moobot/events.py

```diff
@@ -140,16 +141,38 @@ async def send_event_announcement(client: discord.Client, event: MoobloomEvent)
     announcement_channel = get_announcement_channel(client)
 
     message = await announcement_channel.send(embed=build_event_announcement_embed(event))
-    await message.add_reaction(settings.rsvp_yes_emoji)
-    await message.add_reaction(settings.rsvp_maybe_emoji)
-    await message.add_reaction(settings.rsvp_no_emoji)
 
     with Session() as session:
         session.add(event)
         event.announcement_message_id = str(message.id)
         session.commit()
 
 
+async def add_event_rsvp_emojis(client: discord.Client) -> None:
+    with Session() as session:
+        events: list[
+            MoobloomEvent
+        ] = (  # if we really care about the extra api calls we can add a state filter here
```

Comment:
yeah not a huge fan of this adding 1 API call per event on every sync

can we fetch the messages in bulk maybe? or otherwise yeah maybe added state in the DB is the way to go

Reply:
one option is to treat this as part of the `out_of_sync` check, and have new events _start_ as being out of sync. Depending on what you intended for that field to mean, this could be seen as hacky or as elegant; lmk

Reply:
added state tracking in db; apparently to view an entire channel history you have to roll your own pagination code. more hassle than it's worth